service/service.go

```diff
@@ -139,6 +141,10 @@ func New(options ...Option) (Owner, error) {
 		metrics.Freeze()
 	}
 
+	if svc.RuntimeMetrics() == nil {
```

Comment:
We should provide a way to opt-out. Somebody should be able to explicitly say "I don't want to report runtime metrics" (as an option).